test/Interop/Cxx/operators/member-inline-typechecker.swift

```diff
@@ -78,3 +78,13 @@ let _ = derivedConstIterWithUD.pointee
 
 var derivedIntWrapper = DerivedFromLoadableIntWrapperWithUsingDecl()
 derivedIntWrapper += LoadableIntWrapper()
+
+func TestPointeeAvailable()  {
```

Comment:
Minor: let's remove the `func` wrappers to keep this consistent with the rest of the test